OpenCVE CVSS Score (#34401)
demisto/content: Packs/OpenCVE/ReleaseNotes/1_0_4.md

Reviewer: Lets be clear on which value is being taken.

Author: Better?

#### Integrations

##### OpenCVE

- Updated the Docker image to: *demisto/python3:3.10.14.92207*.

> - Added a fix where the CVSS score was sometimes not extracted if found under the **metrics** key. It will be extracted from the **cvss** key in such a case.

Reviewer: ```suggestion
- CVSS score will be extracted from the **cvss** key, insead of **metrics** key.
```
